readme.md

```diff
@@ -49,6 +50,9 @@ grunt.initConfig({
 	}
 });
 
+// You can also use `grunt.loadNpmTasks('grunt-sass');`
+// instead of `require('load-grunt-tasks')(grunt);`
```

Comment:
This is not the place to document how Grunt works.

Reply:
@sindresorhus   
It would be better for other people. You must be able to look from different angles, and not just from your-self. But I don't insist.